virt/virtual_machines/vm_networking/virt-attaching-vm-to-sriov-network.adoc

```diff
@@ -24,7 +24,23 @@ include::modules/nw-sriov-network-attachment.adoc[leveloffset=+1]
 
 include::modules/virt-attaching-vm-to-sriov-network.adoc[leveloffset=+1]
 
+include::modules/virt-configuring-cluster-dpdk.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+* xref:../../../scalability_and_performance/what-huge-pages-do-and-how-they-are-consumed-by-apps.adoc#configuring-huge-pages_huge-pages[Configuring huge pages]
+* xref:../../../virt/virtual_machines/vm_networking/virt-attaching-vm-to-sriov-network.adoc#nw-sriov-configuring-device_virt-attaching-vm-to-sriov-network[Configuring SR-IOV network devices]
```

Comment:
I don't think you need this link because the module is included in this assembly.